polygon/heimdall/store.go

```diff
@@ -61,142 +60,134 @@ type Store interface {
 	CheckpointStore
 }
 
+type RoStore interface {
```

Comment:
I'd like to rename this, because:
1. I prefer to not use shortcuts like "Ro" unless it is a type that is super common as it makes the code cryptic for no good reason.
2. Store should store something, but this only reads. 
3. This interface is very similar to the `reader` interface below it (yes, it uses different types, but conceptually they are similar).
4. Nothing in this interface says that it must be implemented in terms of RoTx (even though it does). In theory the code that uses it might not care about RoTx (which is an implementation detail).

Maybe call this StoreReader?

Reply:
1. I can call it `ReadOnlyStore` - does this feel better? seeing `Ro` does not feel cryptic to me, I feel like `Ro` and `Rw` are know abbreviations in our codebase
2. I dont agree with this - you can read from a store, not just write
3. it is not the same, the `reader` interface accepts a `tx` while the `RoStore` does not - that is the point of this, it abstracts away the `tx` which is passed as a constructor input 
4. that's the point - when you use the constructor `NewRoTxStore` you know you are getting a reader that uses a `Tx` as opposed to say a `NewFileRoStore` (just an example)

Reply:
talked offline - comment was simply to rename `RoStore` to something more explicit